block/src/Pos/Block/Logic/VAR.hs

```diff
@@ -117,6 +148,15 @@ verifyAndApplyBlocks rollback blocks = runExceptT $ do
     lift $ normalizeMempool
     pure hh
   where
+    -- Splits an 'OldestFirst' list of blocks into two lists
```

Comment:
"Spans input into `(a,b)` where `a` is either a single genesis block or a maximum prefix or main blocks"